libcperciva/util/sock.c

```diff
@@ -300,10 +300,11 @@ sock_resolve(const char * addr)
 /**
  * sock_listener(sa):
  * Create a socket, set SO_REUSEADDR, bind it to the socket address ${sa},
- * mark it for listening, and mark it as non-blocking.
+ * set the permission (if not -1), mark it for listening,
```

Comment:
Correction: perhaps something like "set the permission to ${sm} (if it is not -1)"